tests/unit_nocloud/test_hdfs.py

```diff
@@ -0,0 +1,31 @@
+from skyplane.obj_store.hdfs_interface import HDFSFile, HDFSInterface
```

Comment:
Unable to find hdfs_interface module when I try to run this test. Probably have to add the HDFS Interface to the Object Store create function.